core/contractsdk/cpp/src/xchain/trust_operators/trust_operators.h

```diff
@@ -7,18 +7,35 @@ class TrustOperators {
 public:
   TrustOperators(xchain::Context *, const uint32_t);
 
-  bool add(const std::string &left_value, const std::string &right_value,
-           const std::string &output_key);
-  bool sub(const std::string &left_value, const std::string &right_value,
-           const std::string &output_key);
-  bool mul(const std::string &left_value, const std::string &right_value,
-           const std::string &output_key);
+  struct Operand {
+    std::string cipher;     //ciphertext
```

Comment:
could cipher contains invisible chars? If yes, can we use base64 encoded string?

Reply:
ciphertext is base64 encoded string

Reply:
> could cipher contains invisible chars? If yes, can we use base64 encoded string?

ciphertext is base64 encoded string